aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -1399,90 +1415,90 @@ TORCH_IMPL_FUNC(_convert_indices_from_csr_to_coo_structured_cpu)
  * https://github.com/scipy/scipy/blob/8a64c938ddf1ae4c02a08d2c5e38daeb8d061d38/scipy/sparse/sparsetools/csr.h
  * Modified to ensure sorted BSR column indices.
  */
-template <class I, class T>
-void _csr_to_block_csr_cpu_kernel(
-    const I n_row,
-    const I n_col,
-    const I R,
+template <class I, class T, bool compressed_rows>
+void _compressed_to_block_compressed_cpu_kernel(
+    const I n_dim_compressed,
+    const I n_dim_plain,
     const I C,
-    const I* input_crow_indices,
-    const I* input_col_indices,
+    const I P,
+    const I D,
```

Comment:
nit: quite hard to understand what these names mean, compared to other parameters.

Reply:
We have:
C == "block size along compressed dimension"
P == "block size along plain dimension"
D == "number of elements in dense dimensions"

Reply:
I think it would not hurt to change these names and/or add comments, we will definitely forget what these mean once this code is touched again.

Reply:
Added comments.  These functions are small, in particular the second one, while body of the first one is well-commented, so hopefully this kind of naming (that is re-used from original version, that had "R" for block size along rows and "C" for block size along cols) won't cause much of a confusion.